### DISCUSSION TITLE
Schema comments not showing in generated Prisma client

### DISCUSSION THREAD

**Discussion by jsbrain** [prisma/prisma#8008] [Q&A]
According to [the docs](https://www.prisma.io/docs/concepts/components/prisma-schema#comments), the `///` comments are not showing up in the generated Prisma client.

But why? Wouldn't that be awesome? 😀 

I guess there's some good reason for this?! ... could someone please explain?

Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
